tests/api/test_datamap.py

```diff
@@ -0,0 +1,39 @@
+# pylint: disable=missing-docstring, redefined-outer-name
+from typing import Generator
+
+import pytest
+from starlette.testclient import TestClient
+
+from fidesapi import main
+from fidesapi.routes.util import API_PREFIX
+from fidesctl.core.config import FidesctlConfig
+
+
+@pytest.fixture()
+def test_client() -> Generator:
+    """Starlette test client fixture. Easier to use mocks with when testing out API calls"""
+    with TestClient(main.app) as test_client:
+        yield test_client
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "organization_fides_key, expected_status_code",
+    [
+        ("fake_organization", 404),
+        ("default_organization", 200),
```

Comment:
I think I need a second set of eyes here to understand why the second test fails with a 500. The first will pass and the second will fail (flipping them around, either will succeed if in the first position)

Reply:
something similar happened to me in [this PR](https://github.com/ethyca/fides/pull/649) and Thomas put in a temporary fix by moving the test to run before others... not sure if it's the same thing going on here though

Reply:
Thanks for the suggestion here! I ended up taking it out for now and will open up a separate issue to resolve the testing if that works for you @ThomasLaPiana - I think it has something to do with [async testing in FastAPI](https://fastapi.tiangolo.com/advanced/async-tests/) and not sure if I want to dive into that in here. 

Error message is `got Future <Future pending cb=[Protocol._on_waiter_completed()]> attached to a different loop`